src/eckit/utils/RLE.h

```diff
@@ -14,19 +14,22 @@
 #ifndef eckit_RLE_h
 #define eckit_RLE_h
 
-
+#include <chrono>
```

Comment:
need to add `#include <optional>` here if you apply my suggestion or in cpp if you don't.